### PR TITLE
Use "currency conversion fee"

### LINKED ISSUE #9996: Use "currency conversion fee" not "foreign exchange fee"
This issue is a result of this P2 post: pbUcTB-5Uz-p2

**TLDR:** We should strongly consider moving away from using "foreign exchange fee" when referring to the fee itself, and instead use "currency conversion fee." We can still use "foreign exchange" when talking about the current rate at which one currency converts to another. This matches [Stripe's usage](https://docs.stripe.com/currencies/conversions).

Instances of "foreign exchange" in code: 

https://github.com/search?q=repo%3AAutomattic%2Fwoocommerce-payments+%22foreign+exchange%22&type=code

### PR DESCRIPTION
**Pull request by aheckler**
Fixes #9996

#### Changes proposed in this Pull Request

Use "currency conversion fee", not "foreign exchange fee". This is for a couple reasons:

- It matches [Stripe's usage](https://docs.stripe.com/currencies/conversions), where "currency conversion fee" refers to the fee charged for changing funds from one currency to another. "Foreign exchange" refers to the actual rate of exchange/conversion.
- "Currency conversion" is a more accessible term that more accurately describes the situation. 

The changes here should not break anything, as they are entirely string changes.

Other notes:

- I checked Transact Platform Server, but there doesn't seem to be anything worth changing there. All uses of "foreign exchange" are internal, or in comments.
- I did NOT change any uses of `FX` or similar. There are tons of these, and what I'm most concerned about with this PR is merchant-facing content.

### CODE REVIEW

**Comment by botwoo** [comment]
### Test the build 

#### Option 1. Jetpack Beta

- Install and activate [Jetpack Beta](https://jetpack.com/download-jetpack-beta/).
- Use this build by searching for PR number `10002` or branch name `fix-9996` in your-test.site/wp-admin/admin.php?page=jetpack-beta&plugin=woocommerce-payments          

#### Option 2. Jurassic Ninja - available for logged-in A12s

:rocket: [Launch a JN site with this branch](https://jurassic.ninja/create/?jetpack-beta&shortlived&nojetpack&woocommerce&woocommerce-payments-dev-tools&branches.woocommerce-payments=fix-9996) :rocket:

:information_source: Install this [Tampermonkey script](https://github.com/Automattic/woocommerce-payments/tree/develop/bin/wcpay-live-branches) to get more options.

---

Build info: 

- Latest commit: 8576d8aa7b22502e164910bb73e98d8d5a373045
- Build time: 2024-12-19 15:39:27 UTC

_Note: the build is updated when a new commit is pushed to this PR._

File: changelog/fix-9996-currency-conversion-fee-phrasing [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: fix
+
+Use "currency conversion fee", not "foreign exchange fee"
```
Comment (nagpai) [inline changelog/fix-9996-currency-conversion-fee-phrasing]: Optional - make a brief mention of where these string changes would appear. e.g. "Use 'currency conversion fee', not 'foreign exchange fee' at various places including payment timeline, .... .

**Review by nagpai — APPROVED** [review]
Code changes are straightforward string replacements and do not affect any code logic. They look good to me. Snapshots are updated and unit tests pass. I have left a tiny optional suggestion for the changelog since it features on our public release note.

File: changelog/fix-9996-currency-conversion-fee-phrasing (hunk 1 above)
Comment (aheckler) [reply changelog/fix-9996-currency-conversion-fee-phrasing]: Updated changelog to reflect this advice. :)

**Pull request merged**